.github/workflows/update-courses.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Courses Data
+on:
+  workflow_dispatch:
+    inputs:
+      semester:
+        description: 'Semester (eg. FA22)'     
+        required: true
+
+jobs:
+  update-courses:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          cache: 'npm'
+      - name: NPM Clean Install
+        run: npm ci
+      - name: Generate Courses Json
+        run: npm run courses-gen
+      - name: Generate Requirements Json
+        run: npm run req-gen
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
```

Comment:
Used [create-pull-request](github.com/marketplace/actions/create-pull-request) github action. Please review this file!